.github/workflows/triage.yml

```
@@ -135,7 +135,7 @@ jobs:
               content: system "swift", "build"
 
             - label: long build
-              path: Formula/(agda|arangodb|boost|deno|dotnet|emscripten|gcc|ghc|llvm|node|ponyc|rust|swift|texlive|qt)(@[0-9]+)?.rb
+              path: Formula/(agda|arangodb|boost|cmake|deno|dotnet|emscripten|gcc|ghc|llvm|node|pango|ponyc|rust|suite-sparse|swift|texlive|qt)(@[0-9]+)?.rb
```

Just a note here: I was deliberately conservative about adding formulae to this list. In particular, I left out formulae that by themselves (without dependents) would not require the longer timeout.

We can change that approach, of course, but I just wanted to be sure that that's what we want to do here.

---

I'd rather `cmake` was labelled to have `CI-skip-recursive-dependents` to be honest (or better: make test-bot not follow test dependencies recursively - I'll need to look into that at some point).

---

> make test-bot not follow test-only dependencies recursively

I've thought about this too. Should this not be similar to how we don't follow build-only dependencies recursively?

---

Yes, I think so.

---

Agreed with both of you, sounds like we're OK with `cmake` being removed and the rest included.
```suggestion
              path: Formula/(agda|arangodb|boost|deno|dotnet|emscripten|gcc|ghc|llvm|node|pango|ponyc|rust|suite-sparse|swift|texlive|qt)(@[0-9]+)?.rb
```